client/benches/tps/oneshot.rs

```diff
@@ -20,7 +20,7 @@ fn main() {
         flush_guard = Some(flame_layer.flush_on_drop());
 
         tracing_subscriber::registry().with(flame_layer).init();
-        iroha_logger::disable_logger();
+        iroha_logger::disable_global().expect("Logger wasn't set yet");
```

Comment:
I think `disable_global` fails in the opposite way - if the logger was already set

Reply:
It's true. I put this message with "What assumption I _expect_ to be true?" in mind.

This would be clearer:

```rust
iroha_logger::disable_global().expect("Logger should not be set yet");
```

Reply:
Hm, I usually think of `expect` not as "this is why it should never fire", but in terms of "what the user sees when this gets fired", hence the confusion. Your proposed message serves both goals fine I think

Reply:
Updated the message.